src/olympia/devhub/templates/devhub/addons/submit/sidebar.html

```diff
@@ -4,13 +4,23 @@
 {# List of steps: (text, class). A class of 'all' means the step is relevant
    to listed or unlisted addons, a class of 'listed' means it's only relevant
    to listed addons, not unlisted ones. #}
-{% set NAV = ((_('Getting Started'), 'all'),
+{% if waffle.flag('no-prelim-review') %}
+    {% set NAV = ((_('Getting Started'), 'all'),
+              (_('Upload your add-on'), 'all'),
+              (_('Describe your add-on'), 'all'),
+              (_('Add images'), 'listed'),
+              (_('Select a license'), 'listed'),
+              (_('n/a'), 'hidden'),
```

Comment:
I couldn't think of one - step 6 still needs to exist for when the waffle is off so all the code in step 7 would need to go into step 6, etc.  For a short term migration (I hope a week after we flip the waffle and update existing add-ons all the preliminary code can go) it seemed most efficient.  Suggestions welcome though.